src/components/Proposal/Create/SchemeForms/CreateKnownGenericSchemeProposal.tsx

```diff
@@ -89,15 +105,17 @@ class CreateKnownSchemeProposal extends React.Component<IProps, IState> {
       return;
     }
     setSubmitting(false);
-
+ 
+    const ethValue = await this.getBountyEth(values);
```

Comment:
can you add the check for the bounty network here, so that is is more explicit:

```
if (his.props.genericSchemeInfo.specs.name === "Standard Bounties") {
  ethValue = this.getBountyEth(values)
}

Reply:
8fa6cde